src/hooks/__tests__/useNovel.test.ts

```
@@ -78,6 +78,7 @@ describe('useNovel', () => {
   beforeEach(() => {
     jest.clearAllMocks();
     MMKVStorage.clearAll();
+    console.log('--- Starting test ---');
```

ESLint

[error] Unexpected console statement.

(no-console)